grumphp.yml

```diff
@@ -1,61 +0,0 @@
-parameters:
-    phpstan.config: "vendor/phpro/symfony-conventions/phpstan.neon"
-    phpstan.level: 2
-    phpcsfixer2.config: "vendor/phpro/symfony-conventions/.php_cs.dist"
-    phpparser.ignore:
-        - vendor
-        - config
-        - tests/bootstrap.php
-        - src/Kernel.php
-    phpstan.ignore:
-        - "tests/"
-        - "Tests/"
-        - "vendor/"
-        - "var/"
-        - "src/Migrations/"
-        - "spec/"
-grumphp:
-    stop_on_failure: true
-    tasks:
-        paratest:
-            metadata:
-                priority: 100
```

Comment:
we should drop all these priorities : that way they all run in parallel - making it commit faster.
The only place where I still use priorities is e.g. for code coverage or infection.